tools/clean_demo_hosts.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -x
+
+for host in $(oc get baremetalhosts --no-headers | grep '^demo-' | awk '{print $1}')
+do
+    oc delete baremetalhost $host
```

Comment:
Just as an idea, you could put a label on all of them, and just `oc delete baremetalhost -l metalkubedemo=true` or something like that. This certainly gets the job done though.

Reply:
Good idea. See #108